doc/api/domain.md

```diff
@@ -239,16 +239,16 @@ serverDomain.run(() => {
     // req and res are also created in the scope of serverDomain
     // however, we'd prefer to have a separate domain for each request.
     // create it first thing, and add req and res to it.
-    var reqd = domain.create();
+    const reqd = domain.create();
     reqd.add(req);
     reqd.add(res);
     reqd.on('error', (er) => {
-      console.error('Error', er, req.url);
+      console.error(`Error ${er} ${req.url}`);
```

Comment:
Oh. Yes. Sorry.

I shall revert these ones. It seems something like `Error ${util.format(err)}` will be much worth.